architecture/networking/routes.adoc

```diff
@@ -833,3 +833,53 @@ $ oc expose service/<name> --hostname="m.api.openshift.org"
 $ oc expose service/<name> --hostname="kates.net"
 $ oc expose service/<name> --hostname="api.kates.net"
 ----
+
+
+== Support for Kubernetes Ingress objects in Openshift
+
+In an effort to support the Kubernetes method of managing external access to services
+in a cluster an ingress to route controller has been created. This controller watches
+ingress objects and attempts to create one or more routes to satisfy the conditions of
+the Ingress object. For example
+
+----
+kind: Ingress
+apiVersion: extensions/v1beta1
+metadata:
+  name: test
+spec:
+  rules:
+  - host: test.com
+     http:
+       paths:
+       - path: /test
+         backend:
+           serviceName: test-1
+           servicePort: 80
+----
+
+would generate
+
+----
+kind: Route
+apiVersion: route.openshift.io/v1
+metadata:
+  name: test-a34th
```

Comment:
Maybe a callout to say that the name is automatically generated, with the ingress's name as a prefix.

Reply:
Added in the description